data-ingestion/index.ipynb

```diff
@@ -117,7 +117,10 @@
             "outputs": [],
             "source": [
                 "%%bash --out job_id\n",
-                "bacalhau docker run --wait --input-urls http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz:/inputs/train-images-idx3-ubyte.gz ubuntu -- cp -rv /inputs/. /outputs/"
+                "bacalhau docker run \\\n",
+                "    --id-only\n",
```

Comment:
this is missing a \ on the end of the line?

Reply:
yep